master/custom/workers.py

```diff
@@ -144,6 +144,12 @@ def get_workers(settings):
             tags=['linux', 'unix', 'rhel', 'arm', 'arm64', 'aarch64'],
             parallel_tests=40,
         ),
+        cpw(
+            name="diegorusso-aarch64-bigmem",
+            tags=['linux', 'unix', 'ubuntu', 'arm', 'arm64', 'aarch64', 'bigmem'],
+            not_branches=['3.9', '3.10', '3.11', '3.12', '3.13'],
+            parallel_tests=60,
```

Comment:
Just noticing this value, it seems a little extreme for a bigmem builder.  Theoretically, this could call on 3600GB of RAM (60 tests x 60GB each from the `-M60g` option).  In practice there aren't enough bigmem tests to *actually* request that, but this is a big enough number of tests that it becomes likely that multiple (possibly *all*) bigmem tests will be running in parallel.

Reply:
Yes, my bad. Decreased to 4, so we should be good (60*4=240)